cms/templates/studio_xblock_wrapper.html

```diff
@@ -47,7 +47,11 @@
 % endif
 
 <header class="xblock-header xblock-header-${xblock.category}">
```

Comment:
I doubt it that can be done due to the conditionals. The article was a pre-existing thing (and the header). This would be better handled as an accessibility bug as opposed to trying to change it as part of this PR.